src/restaurant_inspections_api/constants.clj

```diff
@@ -6,3 +6,4 @@
 (def production? (env/production?))
 (def db-url (env/get-env-db-url))
 (def csv-files (env/get-csv-files))
+(def *earliest-date* "2013-01-01")
```

Comment:
Are you specifically using earmuffs to signal that this var will be treated as dynamic? Or for another reason?
See http://www.braveclojure.com/zombie-metaphysics/  (Ctrl+f earmuff or dynamic) on the convention for using asteriks around vars. I ask just in case.